app/src/main.tsx

```diff
@@ -1,10 +1,32 @@
-import { StrictMode } from 'react'
-import { createRoot } from 'react-dom/client'
-import App from './App.tsx'
+import {
+    createBrowserRouter, Outlet,
+    RouterProvider,
+} from "react-router-dom"
 import './index.css'
+import {createRoot} from "react-dom/client";
 
-createRoot(document.getElementById('root')!).render(
-  <StrictMode>
-    <App />
-  </StrictMode>,
+import Example from "./Vues/Example.tsx";
+import {StrictMode} from "react";
+
+const router = createBrowserRouter([
+    {
+        path: "/",
+        element: <div><Outlet/></div>,
+        errorElement: <div></div>,
+        children : [
+            {
+                path: "/example",
```

Comment:
Can we import these routes from another file ?
Should be a bit cleaner. 
Also it would restrict the amount of code we have to view in order to add a simple component.